priam/src/test/java/com/netflix/priam/backup/TestBackupVerification.java

```diff
@@ -125,82 +109,23 @@ private void setUp() throws Exception {
-                            Status.FINISHED));
+                    getBackupMetaData(start.plus(i + 1, ChronoUnit.MINUTES), Status.FINISHED));
         }
-        backupStatusMgr.finish(
-                getBackupMetaData(BackupVersion.SNAPSHOT_BACKUP, start, Status.FINISHED));
+        backupStatusMgr.finish(getBackupMetaData(start, Status.FINISHED));
         backupStatusMgr.failed(
-                getBackupMetaData(
-                        BackupVersion.SNAPSHOT_BACKUP,
-                        start.plus(20, ChronoUnit.MINUTES),
-                        Status.FAILED));
+                getBackupMetaData(start.plus(20, ChronoUnit.MINUTES), Status.FAILED));
         for (int i = 0; i < numFakeBackups - 1; i++) {
             backupStatusMgr.finish(
-                    getBackupMetaData(
-                            BackupVersion.SNAPSHOT_META_SERVICE,
-                            start.plus(i + 1, ChronoUnit.MINUTES),
-                            Status.FINISHED));
+                    getBackupMetaData(start.plus(i + 1, ChronoUnit.MINUTES), Status.FINISHED));
         }
-        backupStatusMgr.finish(
-                getBackupMetaData(BackupVersion.SNAPSHOT_META_SERVICE, start, Status.FINISHED));
-    }
-
-    @Test
-    public void verifyBackupVersion1() throws Exception {
-        setUp();
-        // Verify for backup version 1.0
-        Optional<BackupVerificationResult> backupVerificationResultOptinal =
-                backupVerification.verifyLatestBackup(
-                        BackupVersion.SNAPSHOT_BACKUP,
-                        false,
-                        new DateRange(backupDate + "," + backupDate));
-        Assert.assertTrue(backupVerificationResultOptinal.isPresent());
-        Assert.assertEquals(Instant.EPOCH, backupVerificationResultOptinal.get().snapshotInstant);
-        Optional<BackupMetadata> backupMetadata =
-                backupStatusMgr
-                        .getLatestBackupMetadata(
-                                BackupVersion.SNAPSHOT_BACKUP,
-                                new DateRange(backupDate + "," + backupDate))
-                        .stream()
-                        .findFirst();
-        Assert.assertTrue(backupMetadata.isPresent());
-        Assert.assertNotNull(backupMetadata.get().getLastValidated());
-
-        backupMetadata =
-                backupStatusMgr
-                        .getLatestBackupMetadata(
-                                BackupVersion.SNAPSHOT_META_SERVICE,
-                                new DateRange(backupDate + "," + backupDate))
-                        .stream()
-                        .findFirst();
-        Assert.assertTrue(backupMetadata.isPresent());
-        Assert.assertNull(backupMetadata.get().getLastValidated());
+        backupStatusMgr.finish(getBackupMetaData(start, Status.FINISHED));
     }
 
     @Test
     public void verifyBackupVersion1DateRange() throws Exception {
```

Comment:
It will be good if we can update the function name(remove version1) or remove this test completely as v2 test is already present in verifyBackupVersion2DateRange

Reply:
Agreed. The test is redundant and should be removed. I will do that in a follow-up.